tests/unit/test_pulls.py

```diff
@@ -286,3 +286,21 @@ def test_contents(self):
         self.session.get.assert_called_once_with(
             self.example_data['contents_url']
         )
+
+
+class TestPullFilePatch(helper.UnitHelper):
```

Comment:
Seems as though we're just duplicating the test above but with different example data. What if we make this class inherit from that one and just keep L295-L298?